sdk/lib/cpprt/arm/cpprt.s

```diff
@@ -15,9 +15,15 @@ _Target SETS "|$Target|"
     ; void __cdecl `eh vector constructor iterator'(void *,unsigned int,int,void (__cdecl*)(void *),void (__cdecl*)(void *))
     DEFINE_ALIAS ??_L@YAXPAXIHP6AX0@Z1@Z, ?MSVCRTEX_eh_vector_constructor_iterator@@YAXPAXIHP6AX0@Z1@Z
 
+    ; void __cdecl `eh vector constructor iterator'(void *,unsigned int,unsigned int,void (__cdecl*)(void *),void (__cdecl*)(void *))
+    DEFINE_ALIAS ??_L@YAXPAXIIP6AX0@Z1@Z, ?MSVCRTEX_eh_vector_constructor_iterator@@YAXPAXIHP6AX0@Z1@Z
+
     ; void __cdecl `eh vector destructor iterator'(void *,unsigned int,int,void (__cdecl*)(void *))
     DEFINE_ALIAS ??_M@YAXPAXIHP6AX0@Z@Z, ?MSVCRTEX_eh_vector_destructor_iterator@@YAXPAXIHP6AX0@Z@Z
 
+    ; void __cdecl `eh vector destructor iterator'(void *,unsigned int,unsigned int,void (__cdecl*)(void *))
+    DEFINE_ALIAS ??_M@YAXPAXIIP6AX0@Z@Z, ?MSVCRTEX_eh_vector_destructor_iterator@@YAXPAXIHP6AX0@Z@Z
```

Comment:
Are the decorated names correct?

Reply:
I copied what is reported on the build log.This seems to "match" i386/amd64. That's all I know.

Reply:
https://demangler.com/
![Screenshot_20210604-052435](https://user-images.githubusercontent.com/2107452/120707452-4cd52200-c4f5-11eb-92f8-6a46087d519a.png)

Reply:
> https://demangler.com/

Thanks for the link!
It confirms what I used.

> This seems to "match" i386/amd64.

PS:
I wonder why amd64 file sometimes has `__int64`/`__ptr64` and sometimes not. Anyway, that is not this PR.
Nonetheless, amd64 devs may want to double-check that...